build/__sdkStorageConstants.sh

```diff
@@ -3,8 +3,8 @@
 ENABLE_DYNAMIC_INSTALL_KEY='ENABLE_DYNAMIC_INSTALL'
 SDK_STORAGE_BASE_URL_KEY_NAME='ORYX_SDK_STORAGE_BASE_URL'
 DEV_SDK_STORAGE_BASE_URL='https://oryxsdksdev.blob.core.windows.net'
-PROD_SDK_STORAGE_BASE_URL='https://oryxsdks.blob.core.windows.net'
-PROD_SDK_CDN_STORAGE_BASE_URL='https://oryxsdk-cdn.azureedge.net'
+PROD_SDK_STORAGE_BASE_URL='https://oryxsdksprod.blob.core.windows.net'
+PROD_SDK_CDN_STORAGE_BASE_URL='https://oryx-cdn.azureedge.net'
```

Comment:
This should be the DNS url?

Reply:
I see, gonna change it. And just want to confirm `DEV_SDK_STORAGE_BASE_URL` remains the same?

Reply:
yes right. It will remain the same.